target_snowflake/db_sync.py

```diff
@@ -144,15 +144,25 @@ def flatten_schema(d, parent_key=[], sep='__', level=0, max_level=0):
 
     return dict(sorted_items)
 
+def _should_json_dump_value(key, value, flatten_schema=None):
+    if isinstance(value, (dict, list)):
```

Comment:
A fallback `return False` is missing!
I think you can just do `return isinstance(value, (dict, list))`.

Reply:
It is not missing, the code of this function is 
```
def _should_json_dump_value(key, value, flatten_schema=None):
    if isinstance(value, (dict, list)):
        return True

    if flatten_schema and key in flatten_schema and 'type' in flatten_schema[key] and set(flatten_schema[key]['type']) == {'null', 'object', 'array'}:
        return True

    return False
```

I split into two ifs on purpose, since it's a bit easier to read and 2 clear distinct cases handled

Reply:
Oh ok.